script/setup.ts

```diff
@@ -1,8 +1,13 @@
 import { exec } from "child_process";
 import { satisfies } from "semver";
 import { createLogger, format, Logger, transports } from "winston";
+const ora = require("ora");
```

Comment:
```suggestion
import * as ora from "ora";
```

Reply:
Hi @tupe12334, I tried this but this library does not work if I import using import, 
I discussed the same with @abrl91 in the discord thread.
https://discord.com/channels/757179260417867879/971606005672722482

Reply:
@jainpawan21 Please check how I did it and it's supposed to work, please let me know if it didn't work.

Reply:
Thanks It worked 🚀